tests/system/tstrmantle.nim

```diff
@@ -0,0 +1,46 @@
+var res = newStringOfCap(24)
+
+for i in 0 .. 9:
+  res.addInt int64(i)
+
+doAssert res == "0123456789"
+
+res = newStringOfCap(24)
```

Comment:
@xflywind 
to reset, instead use `res.setLen 0` (more efficient since you reuse same buffer, and simpler)

Reply:
done https://github.com/nim-lang/Nim/pull/16235